applications/mne_scan/libs/scDisp/realtime3dwidget.cpp

```diff
@@ -368,7 +362,53 @@ void RealTime3DWidget::alignFiducials(QSharedPointer<FIFFLIB::FiffDigitizerData>
     m_pFiffDigitizerData = pDigData;
 
     if (m_pFiffDigitizerData){
+        MneMshDisplaySurfaceSet* pMneMshDisplaySurfaceSet = new MneMshDisplaySurfaceSet();
```

Comment:
I know it is hard to keep an eye on this while concentrated on making the thing working but, could we use smart pointers? If we build the habit, it should become less hard.

Reply:
This is currently just a copy/paste of the previous allignFiducials() function, but without the requirement of reading from file. If it works(when we test on the neuromag machine), next step is to refactor the calculations to a new function that both the from-file and not-from-file versions can call. In there i think it makes sense to change it.

Reply:
great